### ISSUE TITLE
output PDF is not same as in the HTML

### ISSUE THREAD

**Issue by jessicastorm1986** [MarouaneSH/Ionic-jsPdf-Html2Canvas#1]
Almost less than half of the image is appearing on pdf, have you noticed that ?

**Comment by MarouaneSH** [comment]
yeah i notice that to, because html2canvas capture just he current screen view.
however, you can use this [cordova plugin](https://github.com/cesarvr/pdf-generator) to convert from html to pdf.
